src/TestHelper/MsSqlConnectionBuilder.cs

```diff
@@ -33,7 +33,7 @@ public static void Setup(string tenantId)
             using (var conn = MsSqlConnectionBuilder.Build())
             {
                 conn.Open();
-                conn.ExecuteCommand($"if not exists (select * from sysdatabases where name = '{dbName}') create database {dbName};");
+                conn.ExecuteCommand($"if not exists (select * from sysdatabases where name = '{dbName}') create database {dbName}; ALTER DATABASE [{dbName}] COLLATE SQL_Latin1_General_CP1_CS_AS;");
```

Comment:
What happens on the build agent when the db already exists?

Reply:
The command just executes.

Reply:
I take that back, it only executes if the schema is already case sensitive...